--- a/index.js	2019-07-10 01:03:42.000000000 +0300
+++ b/index.js	2019-07-10 01:03:12.000000000 +0300
@@ -18,7 +18,7 @@
 This isn't an error unless you're trying to use a local gateway. 😁`
 
 // Check if the page was loaded from HTTPS
-const IS_SECURE = self.location.href.startsWith('https')
+const IS_SECURE = (typeof self !== 'undefined') && self.location.href.startsWith('https')
 
 class DiscoverySwarmWeb extends EventEmitter {
   constructor (opts = {}) {
